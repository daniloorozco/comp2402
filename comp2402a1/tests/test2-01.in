Hey
There
I'm
A
Rock
Star
and
I
Don't
Care
Any
Way
Good
Bye
