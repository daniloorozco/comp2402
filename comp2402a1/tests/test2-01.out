Hey
I'm
Rock
and
Don't
Any
Good
There
A
Star
I
Care
Way
Bye
